client/src/components/backend/Dialog/ResetPassword.js

```diff
@@ -213,6 +211,11 @@ class ResetPasswordWrapper extends PureComponent {
 
 }
 
+ResetPasswordWrapper.contextTypes = {
```

Comment:
Can't these be added as static properties on the class? That would be more in keeping with what we're doing elsewhere with propTypes, etc.